Cabal/Distribution/PackageDescription/Parse.hs

```diff
@@ -161,6 +162,16 @@ storeXFieldsPD _ _ = Nothing
 -- ---------------------------------------------------------------------------
 -- The Library type
 
+subLibFieldDescrs :: [FieldDescr Library]
+subLibFieldDescrs =
+  [ simpleField "library"
+                        showToken             parseTokenQ
+                        -- The invariant for printing a 'Library' @lib@ is that
+                        -- @isJust (libName lib)@.
+                        (fromMaybe (error "subLibFieldDescrs") . libName)
+                        (\xs lib -> lib{libName=Just xs})
+  ] ++ libFieldDescrs
+
```

Comment:
I don't understand why this is in this patch. It's just for the pretty-printer apparently. Is this fixing something that was already broken previously? Is this change supposed to live in the patch about the pretty printer?